core/src/main/web/outputdisplay/bko-plot/bko-plot.js

```diff
@@ -482,6 +483,23 @@
           });
         };
 
+        scope.pinCloseIcon = function(d) {
+          var tipid = "tip_" + d.id;
```

Comment:
looks like this code was copied.  instead, make a function that you can call from each place.